src/librustc/mir/interpret/pointer.rs

```diff
@@ -213,20 +213,4 @@ impl<'tcx, Tag> Pointer<Tag> {
     pub fn erase_tag(self) -> Pointer {
         Pointer { alloc_id: self.alloc_id, offset: self.offset, tag: () }
     }
-
-    /// Test if the pointer is "inbounds" of an allocation of the given size.
-    /// A pointer is "inbounds" even if its offset is equal to the size; this is
-    /// a "one-past-the-end" pointer.
-    #[inline(always)]
-    pub fn check_inbounds_alloc(
```

Comment:
This one only had two callers, and one caller did `is_err()` so it didn't even want to raise an error. So I inlined it on both calls. This also means some fewer exceptions being caught, which is always great. :) Cc https://github.com/rust-lang/rust/issues/69297